### PR TITLE
Updated sources to bibtex handles

### PR DESCRIPTION
**Pull request by LeoHeidweiler**
Changed name and location of references.bib to inst/extdata/database/sources.bib
Added new sources to sources.bib
Minor corrections to the files in inst/extdata/database/teds/Tech
Changed all entries in "source" for the files in inst/extdata/database/teds/Tech to the respective bibtex handles

### CODE REVIEW

File: .DS_Store
Comment (PhilippVerpoort) [inline .DS_Store:1]: Ich glaube, diese Datei war ein Versehen. Könntest du die vorm Merge noch rauswerfen?

File: inst/extdata/database/sources.bib [hunk 1]
```diff
@@ -360,6 +386,22 @@ @techreport{IEA-DAC22
...
+	institution = {International Energy Agency},
+	title =  {The Future of Hydrogen}
+	url = {https://www.iea.org/reports/the-future-of-hydrogen}
+	year = {2019}
+}
+
+@techreport{IEA-GHG13,
+	author = {IEA},
+	institution = {International Energy Agency},
+	title = {Iron and Steel CCS Study (Techno-Economics Integrated Steel Mill},
```
Comment (PhilippVerpoort) [inline inst/extdata/database/sources.bib:400]: Hier ist noch ein kleiner Typo (fehlende geschlossene Klammer).

**Review by PhilippVerpoort — CHANGES_REQUESTED** [review]

File: inst/extdata/database/flow_types.csv
Comment (PhilippVerpoort) [inline inst/extdata/database/flow_types.csv:1]: Sorry, just realising one more thing: was there a reason why you removed the unit variant from the default unit in `flow_types.csv`?

**Review by PhilippVerpoort — CHANGES_REQUESTED** [review]

File: inst/extdata/database/flow_types.csv
Comment (LeoHeidweiler) [reply inst/extdata/database/flow_types.csv:1]: My program for viewing tables wrongly interpreted the ; in the unit as a separator, which made the table look like it had mistakes in it. I didn't realize the underlying problem and tried to correct the error. Thanks for the note, I have now reverted the change and restored the file to its original form

Comment (PhilippVerpoort) [reply inst/extdata/database/flow_types.csv:1]: Perfect, thanks!

**Review by PhilippVerpoort — APPROVED** [review]

**Comment by PhilippVerpoort** [comment]
Please go ahead with the merge!

I prefer using rebase or squash to keep the history clean.

**Comment by PhilippVerpoort** [comment]
(Maybe a squash with description copied from individual commits would be preferable, also given the back-and-forth with some of the accidental changes.)

**Pull request merged**